### DISCUSSION TITLE
`ADR Suggestion` Naming of the spectroscopy and imaging modules

### DISCUSSION THREAD

**Discussion by rozyczko** [easyscience/.github#10] [Ideas]
#### View 1:
Naming it `EasySpectroscopy` would probably be a good idea.
Even if we only start with QENS, we can still envision/plan for adding more techniques: like spin or INS.
The naming is straightforward, avoids acronyms, and follows the `Easy<Technique>` pattern of `EasyDiffraction` and `EasyReflectometry`.

#### View 2:
The analysis of QENS data is very different from spin or INS.

With QENS, the model is a set of simple functions.
With spin spectroscopy the model is atomistic.
With INS the model is both atomistic structure and phonon spectrum.

The workflow of EasyApp would get rather muddled and complex if we wanted to allow all the possible models to be defined in a single app.

Also, QENS and INS are well-accepted and understood names of sub-techniques.

## Imaging

Similar discussion needs to be started about imaging.
Here, we are going to provide analysis for Bragg Edge imaging only, which is based on the atomistic structure of the system.

However, this is not the only thing neutron imaging is used for. Programs like MuhRec do tomography reconstruction analysis - a task not suitable for EasyApp workflow (or is it?)

One could claim calling a module `EasyImaging` is therefore misleading, if the only analysis it can do is Bragg Edge fitting.

**Comment by damskii9992** [comment]
Would it be an idea to get inputs from the IDS's on this as well?

**Comment by damskii9992** [comment]
Also, regarding easyBragg, I think I would prefer a slightly more descriptive name, such as easyBraggfit (easyBraggFit?) or easyBraggimaging (easyBraggImaging?), to better specifiy what the software actually does.

**Reply by rozyczko** [reply]
Then maybe naming it `EasyImaging` makes sense? It will only be used for BE fitting so there will not be any feature creep (planned).

**Comment by rozyczko** [comment]
Agreed on: `EasyImaging`

Still in discussions:
EasyQENS
EasyQens

**Comment by damskii9992** [comment]
Just trying to wake this discussion up again, especially considering the work in #20, have there been made a choice regarding the name of the QENS package/repository yet @henrikjacobsenfys?

**Reply by henrikjacobsenfys** [reply]
It's called EasyDynamics at the moment, but as time passes, I'm more and more in favor of EasySpectroscopy. I'll bring it up again next time I talk to the spectroscopy people.

**Reply by damskii9992** [reply]
Hey @henrikjacobsenfys, considering you're getting close to a first minimal release, you should probably make your final decision on the name just about now. Will you be going with EasySpectroscopy or EasyDynamics?

**Comment by rozyczko** [comment]
This should not be an ADR but just a discussion, I think.

**Reply by damskii9992** [reply]
I think it makes sense as an ADR, as the discussion is how we name our projects and could be relevant to future projects. Namely that we decided to use generic technique names rather than descriptive names, to avoid having overly long and complicated names and to allow increasing breadth of functionality in the future without renaming.

**Comment by rozyczko** [comment]
Agreed on closing it forever
